src/main/java/io/appform/dropwizard/discovery/bundle/resolvers/DefaultPortSchemeResolver.java

```diff
@@ -0,0 +1,26 @@
+package io.appform.dropwizard.discovery.bundle.resolvers;
+
+import io.appform.ranger.core.model.PortSchemes;
+import io.dropwizard.jetty.ConnectorFactory;
+import io.dropwizard.jetty.HttpsConnectorFactory;
+import io.dropwizard.server.DefaultServerFactory;
+import io.dropwizard.server.ServerFactory;
+import lombok.val;
+
+public class DefaultPortSchemeResolver implements PortSchemeResolver {
+
+  private String getScheme(ConnectorFactory connectorFactory){
+    if(null == connectorFactory) return PortSchemes.HTTP;
+    return connectorFactory instanceof HttpsConnectorFactory ? PortSchemes.HTTPS : PortSchemes.HTTP;
```

Comment:
There might be other types of ConnectorFactory. It is better to write an if instanceOf chain and throw an IllegalArgumentException so that implementer knows that an override needs to happen